options/locale/locale_de-DE.ini

```diff
@@ -1135,6 +1135,11 @@ diff.data_not_available=Keine Diff-Daten verfügbar
 diff.show_diff_stats=Diff-Statistik anzeigen
 diff.show_split_view=Geteilte Ansicht
 diff.show_unified_view=Gesamtansicht
+diff.whitespace_button=Whitespace
```

Comment:
Thank you for contributing these translations, translations need to be provided by crowdin: https://crowdin.com/project/gitea 

This is feedback about this specific file not a review of the entire PR